content/enterprise_influxdb/v1.9/administration/manage/security/fine-grained-authorization.md

```diff
@@ -28,6 +24,11 @@ FGA does not restrict actions performed by Flux queries (both read and write).
 If using FGA, we recommend [disabling Flux](/enterprise_influxdb/v{{< current-version >}}/flux/installation/).
 {{% /warn %}}
 
+{{% note %}}
+FGA is only available in InfluxDB Enterprise.
+InfluxDB OSS 1.x controls access at the database level only.
+{{% /note %}}
+
```

Comment:
three links on this page go to top-level Manage security page:
- admin permissions
- enable authentication
- User management commands